backend/src/controllers/students.js

```diff
@@ -0,0 +1,29 @@
+const db = require('../db');
+
+const getStudents = async (req, res) => {
```

Comment:
Deze naam is misschien niet enorm informatief. Ik vraag me hierbij af of deze code bijvoorbeeld de studenten van een teacher ophaalt of van een class (als ik naar de code kijk, weet ik dat wel uiteraard, maar misschien kan je dit renamen naar iets van `getStudentsByClass` of zoiets voor de duidelijkheid?). Vooral als je ook een andere endpoint toevoegt die wel de studenten van een teacher ophaalt